Apps/Contoso.UWP.Puppet/MainPage.xaml.cs

```diff
@@ -19,6 +21,13 @@ public MainPage()
         {
             this.InitializeComponent();
             Application.Current.UnhandledException += OnUnhandledException;
+            SetMemoryWarning();
+        }
+
+        private async void SetMemoryWarning()
```

Comment:
I think this can be changed to regular synced method
```suggestion
        private void SetMemoryWarning()
```